api/src/main/java/no/nav/sbl/dialogarena/modiabrukerdialog/api/service/gsak/SakerService.kt

```diff
@@ -0,0 +1,20 @@
+package no.nav.sbl.dialogarena.modiabrukerdialog.api.service.gsak
+
+import no.nav.sbl.dialogarena.modiabrukerdialog.api.domain.gsak.Sak
+import no.nav.sbl.dialogarena.modiabrukerdialog.api.exceptions.JournalforingFeilet
+
+
+interface SakerService {
+    class Resultat(val saker: ArrayList<Sak> = ArrayList(), val feiledeSystemer: ArrayList<String?> = ArrayList())
```

Comment:
To endringer jeg tror kan være kjekt å få inn her;
1: bytte til `List<T> = ArrayList()` da vi ikke trenger å eksponere at vi bruker en ArrayList her.
2. `feiledeSystemer` trenger ikke å godta `null` verdier i listen

Så noe alà; `class Resultat(val saker: List<Sak> = ArrayList(), val feiledeSystemer: List<String> = ArrayList())` kanskje??

Reply:
`In Kotlin, all non-mutable collections, such as List, are compile-time read-only by default, and not immutable. While the defined interfaces do not support methods to change data within the collection, the underlying data can still be changed.`

Jeg kan ikke bruke `list.add `...

Reply:
Alle klasser som implenterer `SakerKilde` feiler fordi at List.add ikke tilatt i kotlin